game-core/build.gradle

```diff
@@ -346,10 +342,6 @@ checkstyleTest {
     source sourceSets.test.output.resourcesDir
 }
 
-compileTestJava {
-    options.compilerArgs += [ '-Xep:ClassCanBeStatic:OFF' ]
```

Comment:
had to remove as build stated this arg was invalid. Seems it was added for warning suppresion
```
commit 1962fe5fc06c35380d5fec314053a96e8b500ac2
Date:   Sat Dec 23 10:52:41 2017 +0100
    Just ignore classcanbestatic warnings for test code
```

Reply:
this turns out to be okay, if we are removing error prone then this can be dropped too.